src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -120,6 +120,32 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             }
         });
 
+        _updateScrollBar = std::make_shared<ThrottledFunc<ScrollBarUpdate>>(
+            [weakThis = get_weak()](const auto& update) {
+                if (auto control{ weakThis.get() })

```

Comment:
And does the following work (...to reduce the nesting level)?
```
if (!auto control{ weakThis.get() }) {
    return;
}
```

Reply:
no unfortunately it doesn't work